src/ugc.rs

```diff
@@ -886,6 +885,12 @@ impl<'a> QueryResults<'a> {
         (0..self.returned_results())
             .map(move |i| self.get(i).unwrap())
     }
+
+    /// Returns an iterator that runs over all the fetched results, but doesn't panic if one of those results failed
+    pub fn iter_maybe<'b>(&'b self) -> impl Iterator<Item=Option<QueryResult>> + 'b {
```

Comment:
I think i'd prefer making the API change here and making this replace `iter`. Panicking from a library isn't a great idea so i'd consider it a good change to make.

Reply:
Agreed. Sorry for the delay - done now.